src/shell/Shell.cpp

```diff
@@ -637,6 +637,17 @@ int main(int argc, char* argv[])
                     context->initDebugger(nullptr);
                     continue;
                 }
+                if (strcmp(argv[i], "--debugger-wait-source") == 0) {
+                    StringRef* sourceName;
+                    clientSourceRef = context->getClientSource(&sourceName);
```

Comment:
`clientSourceRef` is used only for this if statment, so please define this variable inside it.